Sources/Build/BuildDelegate.swift

```diff
@@ -235,6 +236,14 @@ public final class BuildDelegate: BuildSystemDelegate, SwiftCompilerOutputParser
     }
 
     public func commandStatusChanged(_ command: SPMLLBuild.Command, kind: CommandStatusKind) {
+        guard command.shouldShowStatus else { return }
+        guard !swiftParsers.keys.contains(command.name) else { return }
+        guard !isVerbose else { return }
```

Comment:
Makes sense to put this line above the previous one, since it is cheaper to compute (won't matter in practice, but its a good general practice)

Reply:
Agreed. Fixed in next commit.